src/RazorSdk/SourceGenerators/RazorSourceGenerationContext.cs

```diff
@@ -34,13 +31,9 @@ internal class RazorSourceGenerationContext
         public bool WaitForDebugger { get; private set; }
 
         /// <summary>
-        /// Gets a flag that determine if the source generator should log verbose messages.
+        /// Gets a flag that determines if generated Razor views and Pages includes the <c>RazorSourceChecksumAttribute</c>.
         /// </summary>
-        /// <para>
-        /// To configure this using MSBuild, use the <c>_RazorSourceGeneratorLog</c> property.
-        /// For instance <c>dotnet msbuild /p:_RazorSourceGeneratorLog=true</c>
-        /// </para>
-        public bool EnableLogging { get; private set; }
+        public bool GenerateMetadataSourceChecksumAttributes { get; private set; }
```

Comment:
How is this also a replacement for logging ?

Reply:
It's unused